src/components/molecules/plan/plan.scss

```
@@ -0,0 +1,113 @@
+@use "../../variables" as *;
+table {
+  width: 100%;
+}
+
+.plan {
+  margin-bottom: 24px;
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  &:nth-child(even) {
+    .plan-title {
+      background-color: $orange_bistre;
+      border-top: 5px;
+      color: $white;
+      padding: 20px 40px;
+      border-top: solid 5px $orange_bistre;
+      border-left: solid 5px $orange_bistre;
+      border-right: solid 5px $orange_bistre;
+    }
+    .plan-price {
+      border-left: solid 5px $orange_bistre;
+      border-right: solid 5px $orange_bistre;
+    }
+    .plan-membershipfee {
+      border-left: solid 5px $orange_bistre;
+      border-right: solid 5px $orange_bistre;
+    }
+    .plan-weekday {
+      border-left: solid 5px $orange_bistre;
+      border-right: solid 5px $orange_bistre;
+    }
+    .plan-weekends {
+      border-left: solid 5px $orange_bistre;
+      border-right: solid 5px $orange_bistre;
+    }
+    .remarks {
+      border-bottom: solid 5px $orange_bistre;
+      border-left: solid 5px $orange_bistre;
+      border-right: solid 5px $orange_bistre;
+    }
+  }
+}
+
+.plan-title {
+  height: 69px;
+  font-weight: 700;
+  background-color: $orange_zorba;
+  color: $white;
+  font-size: $fontsize-l;
+  padding: 20px 40px;
+  line-height: 29px;
+  border: solid 1px $orange_zorba;
+}
+
+.plan-price {
+  height: 142px;
+  padding: 24px 48px;
+  line-height: 28.96px;
+  font-weight: 700;
+  color: $orange_bistre;
+  font-size: $fontsize-l;
+  background-color: $gray_desertStorm;
+  border: solid 1px $gray_gainsboro;
+}
+
+span.price {
+  line-height: 58px;
+  font-size: $fontsize-xxl;
+  padding-left: 4px;
+}
+span.month {
+  line-height: 20px;
+  font-weight: 400;
+  font-size: $fontsize-s;
+  color: $black;
+}

```

```suggestion
span.month {
  line-height: 20px;
  font-weight: 400;
  font-size: $fontsize-s;
  color: $black;
  display: block;
}
```
> 5000と月額の間を16pxにするために、display:inline-block;をきかせbrを消したつもりなのですが、改行できませんでした

こうすると、monthがブロック要素となり改行されるかと！